server/src/services/media.service.ts

```diff
@@ -65,7 +65,12 @@ export class MediaService extends BaseService {
       const jobs: JobItem[] = [];
 
       for (const asset of assets) {
-        const { previewFile, thumbnailFile } = getAssetFiles(asset.files);
+        if (!(asset as Asset).files) {
```

Comment:
jobs should definitely not need casting, especially to a type used by the mapper.